config/stored_requests.go

```diff
@@ -190,12 +190,15 @@ func (cfg *DatabaseConfig) validate(dataType DataType, errs []error) []error {
 // DatabaseConnection has options which put types to the Database Connection string. See:
 // https://godoc.org/github.com/lib/pq#hdr-Connection_String_Parameters
 type DatabaseConnection struct {
```

Comment:
@VeronikaSolovei9 were you able to test mysql and postgres with and without TLS locally?

Reply:
I tested it with local MySQL server and Production Postgres. Both without TLS configs